core/app/models/schedule.rb

```diff
@@ -18,10 +19,33 @@ def self.from_sections sections
     end
   end
 
+  def find_sort_values
```

Comment:
I think this should be split into two methods: `average_state`, and `average_finish`. This is more idiomatic, and will make it easier to reuse the attributes elsewhere

Reply:
Okay cool.